internal/tools/securitypolicy/main.go

```diff
@@ -244,51 +242,52 @@ func validateEnvRules(rules []EnvironmentVariableRule) error {
 	return nil
 }
 
-func convertCommand(toml []string) map[string]string {
+func convertCommand(toml []string) securitypolicy.CommandArgs {
 	json := map[string]string{}
 
 	for i, arg := range toml {
 		json[strconv.Itoa(i)] = arg
 	}
 
-	return json
+	return securitypolicy.CommandArgs{
+		Elements: json,
+	}
 }
 
-func convertEnvironmentVariableRules(toml []EnvironmentVariableRule) map[string]sp.SecurityPolicyEnvironmentVariableRule {
-	json := map[string]sp.SecurityPolicyEnvironmentVariableRule{}
+func convertEnvironmentVariableRules(toml []EnvironmentVariableRule) securitypolicy.EnvRules {
+	json := map[string]securitypolicy.EnvRule{}
 
 	for i, rule := range toml {
-		jsonRule := sp.SecurityPolicyEnvironmentVariableRule{
+		jsonRule := securitypolicy.EnvRule{
 			Strategy: rule.Strategy,
 			Rule:     rule.Rule,
 		}
 
 		json[strconv.Itoa(i)] = jsonRule
 	}
 
-	return json
+	return securitypolicy.EnvRules{
+		Elements: json,
+	}
 }
 
-func addContainer(containers map[string]sp.SecurityPolicyContainer, container sp.SecurityPolicyContainer) map[string]sp.SecurityPolicyContainer {
-	index := strconv.Itoa(len(containers))
+func addContainer(containers *securitypolicy.Containers,
+	container securitypolicy.Container) {
```

Comment:
Same question about wrapping the second param to a new line. This doesn't seem too long all on one line

Reply:
what is the limit in this codebase, its over 80 here.

Reply:
fyi, this has been changed.